tests/test_market_aggregator.py

```diff
@@ -0,0 +1,91 @@
+import logging
+import os
+from time import sleep
+from d3a_api_client.aggregator import Aggregator
+from d3a_api_client.utils import get_area_uuid_from_area_name_and_collaboration_id
+from d3a_api_client.rest_market import RestMarketClient
+
+
+class TestAggregator(Aggregator):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.is_finished = False
+        self.fee_cents_kwh = 0
+
+    def on_market_cycle(self, market_info):
+        """
+        Places a bid or an offer whenever a new market is created. The amount of energy
```

Comment:
Can you also add offer/bid sample as doc string says?

Reply:
ah, no. This is a remnant of copy/paste. Will adapt the doc string. Of course one can not place bid an offers as a market aggregator.